Code/server/Game/Animation/AnimationEventLists.cpp

```diff
@@ -0,0 +1,188 @@
+#include "AnimationEventLists.h"
+
+/*
+/* The lists in this file might not contain all relevant animation events, so extend as necessary
+*/
+
+const Map<String, String> AnimationEventLists::g_actionsIdleToInstant = {
```

Comment:
While a map lookup is "fast" a constexpr arr lookup may be faster: struct Pair { std::string_view from, to; };

// Keep sorted by `from` (lexicographically) and unique
inline constexpr std::array<Pair, /*N*/> kIdleToInstant = {{

Reply:
Also, avoid String, use std::string_view or a const_char* ptr, the data lives in .rdata anyway

Reply:
we used KSomethingSomething for konstant globals.